net-misc/bopm/bopm-3.1.3-r2.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit eutils multilib user
+
+DESCRIPTION="Blitzed Open Proxy Monitor"
+HOMEPAGE="http://www.blitzed.org/bopm/"
+SRC_URI="http://static.blitzed.org/www.blitzed.org/${PN}/files/${P}.tar.gz"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~ppc ~x86"
+IUSE=""
+
+RDEPEND=""
+DEPEND="${RDEPEND}"
+
+DOCS=(ChangeLog INSTALL README TODO)
+PATCHES=("${FILESDIR}"/remove-njabl.patch)
+
+src_configure() {
+	econf \
+		--datadir="${EPREFIX}"/usr/share/doc/${PF} \
+		--localstatedir="${EPREFIX}"/var/log/${PN}
+}
+
+src_install () {
+	sed -i \
+		-e "s!/some/path/bopm.pid!/run/${PN}/${PN}.pid!" \
+		-e "s!/some/path/scan.log!/var/log/${PN}/scan.log!" \
+		bopm.conf.sample || die
+
+	# Custom Makefile.am rules do not respect DESTDIR,
+	# thus override sysconfdir and localstatedir.
+	emake \
```

Comment:
I don't think it's about honouring `DESTDIR`, but rather because he wants to specify `$localstatedir`. @shentino try just dropping the whole `emake` call and replacing it with `default`, as `localstatedir` has already been passed via `econf` and should already be in the Makefile. Test whether the resulting layout is the same (which is my guess).